docs/user-guide/command-line-interface.md

```diff
@@ -122,18 +122,22 @@ Examples:
 
 #### `--ext`
 
-This option allows you to specify which file extensions ESLint will use when searching for JavaScript files. By default, it uses `.js` as the only file extension.
+This option allows you to specify which file extensions ESLint will use when searching for JavaScript files in the directories you specify.
+By default, it uses `.js` as the only file extension.
 
 Examples:
 
     # Use only .js2 extension
-    eslint --ext .js2
+    eslint . --ext .js2
 
     # Use both .js and .js2
-    eslint --ext .js --ext .js2
+    eslint . --ext .js --ext .js2
 
     # Also use both .js and .js2
-    eslint --ext .js,.js2
+    eslint . --ext .js,.js2
+
+**Note:** This option has no effect on glob parameters.
+For example, `eslint lib/* --ext .js` will match all files within the `lib/` directory, regardless of extension.
```

Comment:
I'm not sure this is very clear (if you don't know globs). I wonder if it would be better worded in the negative: "Note that even if you specify an extension with `--ext .js`, you still need to specify one or more files, directories, or glob patterns to lint."